src/AndersonAcceleration.f90

```diff
@@ -72,183 +67,239 @@ MODULE AndersonAccelerationTypes
     PROCEDURE,PASS :: reset => reset_AndersonAccelerationType
 ENDTYPE AndersonAccelerationType
 
-!> Exception Handler for use in Anderson Acceleration
-TYPE(ExceptionHandlerType),SAVE :: eAndersonAccelerationType
-
 !> Name of module
-CHARACTER(LEN=*),PARAMETER :: modName='ANDERSONACCELERATIONTYPE'
+CHARACTER(LEN=*),PARAMETER :: modName='AndersonAccelerationTypes'
```

Comment:
why this change?  I'm not exactly opposed to it, but out of Futility, there are only three that are not full caps.

We need them to be consistent (so those other ones will need to change to all caps too) or we change all to camelcase.

Reply:
I'm used to seeing camelcase on the MPACT side, so when I added the missing s on the end of the mod name I switched it over. I don't REALLY have a preference, but the logical choice to me would be to be consistent with MPACT.